controller/viewcontroller.php

```diff
@@ -65,10 +99,10 @@ public function __construct($appName, IRequest $request,
 	 */
 	public function index() {
 		$runningOn = $this->config->getSystemValue('version');
-		$runningOnServer91OrLater = version_compare($runningOn, '9.1', '>=');
+		$runningOnNextcloud10OrLater = version_compare($runningOn, '9.1', '>=');
```

Comment:
I think this can be kept as is otherwise you'll have trouble rebasing each time.